backend/src/main/java/com/team4/backend/repository/MonitorRepository.java

```diff
@@ -0,0 +1,12 @@
+package com.team4.backend.repository;
+
+import com.team4.backend.model.Monitor;
+import org.springframework.data.mongodb.repository.ReactiveMongoRepository;
+import org.springframework.stereotype.Repository;
+import reactor.core.publisher.Mono;
+
+@Repository
+public interface MonitorRepository extends ReactiveMongoRepository<Monitor,String> {
+    Mono<Monitor> findByEmailAndIsEnabledTrue(String email);
+    Mono<Monitor> findByEmail(String email);
```

Comment:
Is this method really needed ? We already have the method findByEmailAndIsEnabledTrue

Reply:
https://github.com/0xD34DC0DE/ProjetIntegreTeam4/pull/7/commits/df28324502e63a3e995c893ccffddfdb3943c62d